devtools-ci-index.yaml

```diff
@@ -875,7 +876,7 @@
             ssh_cmd="ssh $sshopts $CICO_hostname"
             $ssh_cmd yum -y install rsync
             rsync -e "ssh $sshopts" -Ha $(pwd)/ $CICO_hostname:payload
-            $ssh_cmd -t "cd payload/ee_tests && /bin/bash cico_run_EE_tests.sh https://{test_url}"
+            $ssh_cmd -t "cd payload/ee_tests && /bin/bash cico_run_EE_tests.sh https://{test_url} ${BUILD_NUMBER}"
```

Comment:
This will fail because `${anything}` will be recognised as a placeholder for templating. Please double the parentheses i.e. `${{BUILD_NUMBER}}` to fix it

See https://ci.centos.org/view/Devtools/job/devtools-almighty-jobs-jjb-test/6/console

> BUILD_NUMBER parameter missing